hedyorg/hedy
# [BUG FIX] Fixes issue where classes overview would crash on unfinished quiz scores

**Description**
In this PR we fix an issue where the class overview would crash if there was a student that has started any quizzes but has never finished one. This resulted in an attempt to take the `max()` of an empty list: resulting in an error.

**Fixes**
No related issue.

**How to test**
Easiest would be to deploy to production and verify that it doesn't occur anymore.
